src/libvirtApi/snapshot.js

```diff
@@ -31,7 +31,11 @@ import { logDebug } from '../helpers.js';
 
 export function snapshotCreate({ connectionName, vmId, name, description, memoryPath, disks, isExternal, storagePools }) {
     // that flag ought to be implicit for non-running VMs, see https://issues.redhat.com/browse/RHEL-22797
-    const flags = (!isExternal || memoryPath) ? 0 : Enum.VIR_DOMAIN_SNAPSHOT_CREATE_DISK_ONLY;
+    let flags = 0;
+    if (isExternal && !memoryPath)
+        flags |= Enum.VIR_DOMAIN_SNAPSHOT_CREATE_DISK_ONLY | Enum.VIR_DOMAIN_SNAPSHOT_CREATE_QUIESCE;
```

Comment:
These 2 added lines are not executed by any test.

Reply:
IMHO this belongs into PR #1560 -- this is the same functionality, with just some tweaks.